src/client/js/formSentiment.js

```diff
@@ -0,0 +1,53 @@
+function handleSentiment(event) {
+    event.preventDefault();
+
+    // check what text was put into the form field
+    const txt = document.getElementById('text').value;
+    const language = document.getElementById('sentimentLanguage').value;
+    const resultContainer = document.querySelector('.sentiment.result');
+    const spinnerContainer = document.getElementById('spinner');
+
+    document.querySelectorAll('.result').forEach(it => {
+        it.style.display = 'none';
+    });
+    spinnerContainer.style.display = 'inherit';
+
+    console.log('::: Form Submitted :::');
+    fetch('http://localhost:8081/sentiment', {
+        method: 'POST',
+        credentials: 'same-origin',
+        headers: {
+            'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+            txt: escape(txt),
+            language: language
+        })
+    })
+        .then(res => {
+            console.log(res);
+            return res.json();
+        })
+        .then(function (res) {
+            //{"agreement":"AGREEMENT","subjectivity":"SUBJECTIVE","confidence":"92","irony":"NONIRONIC"}
```

Comment:
Commented out code shouldn't be in the committed files. Minor issue, but please fix it.

Reply:
... I forgot it. I just needed it to be reminded of the fields.